src/utils/i18n.ts

```diff
@@ -0,0 +1,9 @@
+export function formatAsCurrency(
```

Comment:
how about adding a block comment to clarify the order of the parameters or wrap the parameters in a {}?

Reply:
Is the signature not enough to communicate the interface?